lib/pages/home_screen/movie_list_widget.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import '/pages/movie_info_screen/app_movie_info.dart';
+import '/pages/home_screen/movie_list_card.dart';
+import '/repository/movie_repository.dart';
+import '/model/movie_data_model.dart';
+
+class MovieListScreen extends StatefulWidget {
+  const MovieListScreen({super.key});
+
+  @override
+  State<MovieListScreen> createState() => _MovieListScreenState();
+}
+
+class _MovieListScreenState extends State<MovieListScreen> {
+  MovieRepository movieRepository = MovieRepository();
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+        future: movieRepository.readJsonData(),
+        builder: (context, data) {
```

Comment:
Add a suggestion add the type of the parameter and you forgot to add a trailing comma